main.cpp

```diff
@@ -0,0 +1,419 @@
+				if (turn == true) {
+					mouse = _window.mapPixelToCoords(sf::Mouse::getPosition(_window));
+
+					for (int i = 0; i < 8; i++) {
+						for (int j = 0; j < 8; j++) {
+							if (sqrs[i][j]->Isclicked(mouse)) {
+								if (selected == nullptr) {
+									selected = sqrs[i][j];
+									std::cout << "clicked coords: " << selected->getPositionx() << "|" << selected->getPositiony() << std::endl;
+									selected->setMark(true);
+								}//pierwsze klikniecie
+
+								else if (selected != nullptr){
+									target = sqrs[i][j];
+
+									if (selected != target) {
+										
+										if (move(selected, target, socket, serveradress, serverport, pack)) {
+											std::cout << "made a move from  " << selected->getPositionx() << "|" << selected->getPositiony() << " to " << target->getPositionx() << "|" << target->getPositiony() << std::endl;
+											selected->setMark(false);
+											selected = nullptr;
+											target = nullptr;
+											turn = false;
+											
+										}
+										else {
+											
+											selected->setMark(false);
+											selected = nullptr;
+											target = nullptr;
+											std::cout << "wrong move!" << std::endl;
+										}
+										
+									}
+									
+								}//drugie klikniecie
+
+							}//is clicked
+
+						}
+					}
+				}
+				
+				
+			}//lewy
+			if (sf::Mouse::isButtonPressed(sf::Mouse::Right)) {
+				if (selected != nullptr) {
+					selected->setMark(false);
+					selected = nullptr;
+				}
+
+			}//prawy
+
+
+
+
+		}//pollevent
+
+		//UPDATE
+		if (socket.isBlocking())
+			socket.setBlocking(false);
+
+		
+		pack.clear();
+		if (socket.receive(pack, serveradress, serverport) == sf::Socket::Done) {
+			pack >> id >> tx >> ty;
+			ctx = static_cast<int>(tx);
+			cty = static_cast<int>(ty);
+			for (int i = 0; i < 8; i++) {
+				for (int j = 0; j < 8; j++) {
+					if (sqrs[i][j]->pce != nullptr && sqrs[i][j]->pce->getID() == id) {
+						sqrs[ctx / 100][cty / 100]->bindPiece(sqrs[i][j]->pce);
+						sqrs[ctx / 100][cty / 100]->pce->SetPosition(sqrs[ctx / 100][cty / 100]->getPositionx(), sqrs[ctx / 100][cty / 100]->getPositiony());
+						sqrs[i][j]->pce = nullptr;
+					}//if
+				}
+			}
+		}
+
+
+		
+		
+
+
+
+//rysowa�sko
+		_window.clear(sf::Color::Red);
+
+
+		//rysowanie pol
+		for (int i = 0; i < 8; i++) {
+			for (int j = 0; j < 8; j++) {
+				_window.draw(*sqrs[i][j]);
+			}
+		}
+		//rysowanie bierek
+		for (int i = 0; i < 32; i++) {
+			_window.draw(*pcs[i]);
+		}
+
+		_window.display();
+	}//isopen
+
+	return 0;
+}
+
+
+bool move(Square* slctd, Square* trgt, sf::UdpSocket &sckt, sf::IpAddress &IP, unsigned short &PORT, sf::Packet &pck) {
+	sckt.setBlocking(true);
+	if (slctd->pce == nullptr)
+		return (false);
+
+	pck.clear();
+	pck << slctd->getPositionx() << slctd->getPositiony() << trgt->getPositionx() << trgt->getPositiony();
+	sf::IpAddress a;// do zapchania
+	unsigned short b;// do zapchania
+	bool moveValuability;
```

Comment:
Na pewno valuability? [valuability -> wartościowość]
Jeżeli dobrze rozumiem, że chodzi o to, czy ruch jest możliwy, to lepiej "isMovePossible"